.github/workflows/smoke_test.yml

```diff
@@ -30,6 +30,8 @@ jobs:
           java-version: '${{ inputs.java-version }}'
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v4
+        with:
```

Comment:
cache was getting in the way I think of the self hosted runner, so disable to hopefully magically fix